site/content/docs/0.0/utilities/z-index.md

```diff
@@ -6,4 +6,44 @@ group: utilities
 toc: true
 ---
 
-{{< callout-soon "page" >}}
+## Example
+
+Use `z-index` utilities to stack elements on top of one another. Requires a `position` value other than `static`, which can be set with custom styles<!-- or using our [position utilities]({{< docsref "/utilities/position/" >}})-->.
+
+{{< callout >}}
+We call these "low-level" `z-index` utilities because of their default values of `-1` through `3`, which we use for the layout of overlapping components. <!--High-level `z-index` values are used for overlay components like modals and tooltips.-->
+{{< /callout >}}
+
+{{< example class="bd-example-zindex-levels position-relative" >}}
+<div class="z-3 position-absolute p-5 rounded-3"><span>z-3</span></div>
```

Comment:
`rounded-3` classes should be removed and mentioned in the main issue instead until these classes exist.

Reply:
I'm fine with what you just said, I'm removing it for sure. Do you think that we should be removing all the `.p-*` and `.m-*` classes as well until we backport them ?

Reply:
No, that's OK for `.p-*` and `.m-*`, we're sure there will be there